tjreports/site/views/reports/view.html.php

```diff
@@ -139,25 +145,26 @@ protected function addToolbar()
 	 */
 	protected function addDocumentHeaderData()
 	{
-		$app = JFactory::getApplication();
-		JHtml::_('formbehavior.chosen', 'select');
-		$document = JFactory::getDocument();
+		$app = Factory::getApplication();
+		HTMLHelper::_('formbehavior.chosen', 'select');
+		$document = Factory::getDocument();
 
-		$com_params	= JComponentHelper::getParams('com_tjreports');
+		$com_params	= ComponentHelper::getParams('com_tjreports');
 		$bootstrapSetting = $com_params->get('bootstrap_setting', 1);
 
 		if (($bootstrapSetting == 3)
 			|| ( $app->isAdmin() && $bootstrapSetting == 1 )
 			|| ( !$app->isAdmin() && $bootstrapSetting == 2 ) )
 		{
-			$document->addStylesheet(JURI::root(true) . '/media/techjoomla_strapper/bs3/css/bootstrap.min.css');
+			$document->addStylesheet(Uri::root(true) . '/media/techjoomla_strapper/bs3/css/bootstrap.min.css');
 		}
 
-		$document->addScript(JURI::root() . '/components/com_tjreports/assets/js/tjrContentService.js');
-		$document->addScript(JURI::root() . '/components/com_tjreports/assets/js/tjrContentUI.js');
-		$document->addStylesheet(JURI::root() . '/components/com_tjreports/assets/css/tjreports.css');
-		$document->addScriptDeclaration('tjrContentUI.base_url = "' . Juri::base() . '"');
-		$document->addScriptDeclaration('tjrContentUI.root_url = "' . Juri::root() . '"');
+		$document->addScript(Uri::root() . '/components/com_tjreports/assets/js/tjrContentService.min.js');
```

Comment:
@pranotiTechjoomla merge this with other script and load this from a single function.